libs/mrwebrtc-unityplugin/include/api.h

```diff
@@ -49,6 +50,10 @@ mrsNativeRenderer_EnableRemoteVideo(mrsNativeVideoHandle nativeVideoHandle,
                                     TextureDesc textures[],
                                     int textureCount) noexcept;
 
+MR_UNITYPLUGIN__API mrsResult MR_UNITYPLUGIN__CALL
+mrsNativeRenderer_UpdateTextures(mrsNativeVideoHandle nativeVideoHandle,
+                                 TextureDesc textures[]) noexcept;
```

Comment:
Is it worth passing an array size, or are we expected always exactly 3 textures? If the latter, can you please add a comment?

Reply:
Ah that's even more confusing because `mrsNativeRenderer_EnableRemoteVideo()` does have a `textureCount`.

Reply:
I believe EnableRemoteVideo was set up like this in anticipation of RGBA implementation having a different texture count. UpdateTextures will probably need textureCount added when RGBA is supported.

Reply:
I ended up working this a bit so EnableRemoteVideo native method no longer deals with setting textures it only subscribes and now only UpdateRemoteTextures sets textures. The functionality was essentially a duplication.